custom_components/dwains_dashboard/js/.gitignore

```diff
@@ -1,2 +1 @@
-node_modules/
-src/
```

Comment:
why remove this?

Reply:
I left just `node_modules` in gitignore because they are populated from npm, but the source directory _should_ be committed to git, otherwise it's impossible to check what to code does or modify the component using just the bundle file. In fact, it's difficult to understand what this PR changes given that the `src` dir hasn't been committed before 😅